### PR TITLE
Feed back for hobbit

### PR DESCRIPTION
**Pull request by Taamir-Ransome**
Requesting feedback for hobbit.

### CODE REVIEW

File: lib/hobbit.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+
```
Comment (r-stemmle) [inline lib/hobbit.rb]: @Taamir-Ransome as a side note, the term "initial commit" should be reserved for the first commit in a repo

Comment (r-stemmle) [reply lib/hobbit.rb]: better name for this commit would be "Create hobbit class"

File: spec/hobbit_spec.rb [hunk 2]
```diff
@@ -58,21 +58,36 @@
...
+    101.times do
+      hobbit.celebrate_birthday
+    end
     # have hobbit age 101 years
+    expect(hobbit.old_hobbit?).to be true
     # check that hobbit.old? returns true
+    hobbit.celebrate_birthday
+
+    expect(hobbit.old_hobbit?).to be true
   end
```
Comment (r-stemmle) [inline spec/hobbit_spec.rb:73]: @Taamir-Ransome most of the commits so far have been atomic and related well to the commit message. In this case, adding a test for age in this commit has nothing to do with the method for ring. See if you can use lazygit to move this work to the proper commit

File: spec/hobbit_spec.rb [hunk 3]
```diff
@@ -58,21 +58,36 @@
     expect(hobbit.adult?).to be true
   end
 
-  xit 'is old at the age of 101' do
-    # create a hobbit
+  it 'is old at the age of 101' do
+    hobbit = Hobbit.new('Stemmle')
+
```
Comment (r-stemmle) [inline spec/hobbit_spec.rb:63]: @Taamir-Ransome This test does not show the age of hobbit. We can see that the hobbit is celebrating birthdays, but this test should should assert the age to give us a warm fuzzy feeling that things are working properly...
```
hobbit = Hobbit.new('Stemmle')
expect(hobbit.age).to eq 0
expect(hobbit.old?)to be false
100.times { hobbit.celebrate_birthday }
expect(hobbit.old?)to be false
hobbit.celebrate_birthdate
expect(hobbit.age).to eq 101
expect(hobbit.old?)to be true
```

**Comment by r-stemmle** [comment]
@Taamir-Ransome Overall good work here. See if you can cleanup the commits using lazygit. Also see if you can change the base branch so you are pulling into main

**Comment by Taamir-Ransome** [comment]
Worked with Stemmle to clean up the trash

**Pull request closed without merging**